src/status_im/contexts/wallet/account/bridge_send/style.cljs

```diff
@@ -0,0 +1,4 @@
+(ns status-im.contexts.wallet.account.bridge-send.style)
+
+(def bridge-send-wrapper
+  {:height "100%"})
```

Comment:
does `flex 1` work here instead? 🤔 
I think we generally avoid percentages on view widths/heights in the codebase